As suggested by @dmlloyd it seems something is off: running manually (in my local folder):
```
$ quarkus build -e -B --clean -Dproperty=value1 -Dproperty2=value2 -Dmaven.repo.local=/home/forked_franz/.m2/repository --cli-test --verbose --cli-test-dir /home/forked_franz/IdeaProjects/quarkus/devtools/cli/target/test-project/CliProjectJBangTest/code-with-quarkus
```
it reports:
```
jbang] [2:588] [ERROR] Issue running postBuild()
dev.jbang.cli.ExitException: Issue running postBuild()
        at dev.jbang.spi.IntegrationManager.runIntegrations(IntegrationManager.java:111)
        at dev.jbang.source.buildsteps.IntegrationBuildStep.build(IntegrationBuildStep.java:38)
        at dev.jbang.source.buildsteps.IntegrationBuildStep.build(IntegrationBuildStep.java:19)
        at dev.jbang.source.AppBuilder.build(AppBuilder.java:98)
        at dev.jbang.source.AppBuilder.build(AppBuilder.java:22)
        at dev.jbang.cli.Build.doCall(Build.java:21)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:145)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:21)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
        at dev.jbang.cli.JBang$3.handle(JBang.java:148)
        at dev.jbang.cli.JBang$3.handle(JBang.java:143)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
        at picocli.CommandLine.execute(CommandLine.java:2170)
        at dev.jbang.Main.main(Main.java:14)
Caused by: java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:569)
        at dev.jbang.spi.IntegrationManager.runIntegrationEmbedded(IntegrationManager.java:182)
        at dev.jbang.spi.IntegrationManager.runIntegrations(IntegrationManager.java:99)
        ... 17 more
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at io.quarkus.launcher.JBangIntegration.postBuild(JBangIntegration.java:133)
        ... 23 more
Caused by: java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:569)
        at io.quarkus.launcher.JBangIntegration.postBuild(JBangIntegration.java:124)
        ... 23 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.jboss.threads.Messages
        at org.jboss.threads.JBossThread.setUncaughtExceptionHandler(JBossThread.java:567)
        at org.jboss.threads.JBossThreadFactory.createThread(JBossThreadFactory.java:99)
        at org.jboss.threads.JBossThreadFactory$ThreadCreateAction.run(JBossThreadFactory.java:83)
        at org.jboss.threads.JBossThreadFactory$ThreadCreateAction.run(JBossThreadFactory.java:75)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at org.jboss.threads.JBossThreadFactory.newThread(JBossThreadFactory.java:69)
        at org.jboss.threads.EnhancedQueueExecutor.<init>(EnhancedQueueExecutor.java:423)
        at org.jboss.threads.EnhancedQueueExecutor$Builder.build(EnhancedQueueExecutor.java:799)
        at io.quarkus.builder.Execution.<init>(Execution.java:65)
        at io.quarkus.builder.BuildExecutionBuilder.execute(BuildExecutionBuilder.java:79)
        at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:161)
        at io.quarkus.deployment.jbang.JBangAugmentorImpl.accept(JBangAugmentorImpl.java:112)
        at io.quarkus.deployment.jbang.JBangAugmentorImpl.accept(JBangAugmentorImpl.java:36)
        at io.quarkus.bootstrap.app.CuratedApplication.runInCl(CuratedApplication.java:138)
        at io.quarkus.bootstrap.app.CuratedApplication.runInAugmentClassLoader(CuratedApplication.java:93)
        at io.quarkus.bootstrap.jbang.JBangBuilderImpl.postBuild(JBangBuilderImpl.java:99)
        ... 28 more
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoSuchMethodError: 'java.lang.Object org.jboss.logging.Logger.getMessageLogger(java.lang.invoke.MethodHandles$Lookup, java.lang.Class, java.lang.String)' [in thread "main"]
        at org.jboss.threads.Messages.<clinit>(Messages.java:19)
        at org.jboss.threads.Version.<clinit>(Version.java:37)
        at org.jboss.threads.EnhancedQueueExecutor.<clinit>(EnhancedQueueExecutor.java:76)
        at org.jboss.threads.EnhancedQueueExecutor$Builder.<init>(EnhancedQueueExecutor.java:485)
        at io.quarkus.builder.Execution.<init>(Execution.java:57)
        ... 35 more
```

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -113,7 +113,7 @@
         <wildfly-client-config.version>1.0.1.Final</wildfly-client-config.version>
         <wildfly-elytron.version>2.6.0.Final</wildfly-elytron.version>
         <jboss-marshalling.version>2.2.1.Final</jboss-marshalling.version>
-        <jboss-threads.version>3.6.1.Final</jboss-threads.version>
+        <jboss-threads.version>3.8.0.Final</jboss-threads.version>
         <vertx.version>4.5.10</vertx.version>
         <httpclient.version>4.5.14</httpclient.version>
         <httpcore.version>4.4.16</httpcore.version>

--- a/core/builder/src/main/java/io/quarkus/builder/Execution.java
+++ b/core/builder/src/main/java/io/quarkus/builder/Execution.java
@@ -56,6 +56,7 @@ final class Execution {
         this.finalIds = finalIds;
         final EnhancedQueueExecutor.Builder executorBuilder = new EnhancedQueueExecutor.Builder();
         executorBuilder.setRegisterMBean(false);
+        executorBuilder.setQueueLimited(false);
         executorBuilder.setCorePoolSize(8).setMaximumPoolSize(1024);
         executorBuilder.setExceptionHandler(JBossExecutors.loggingExceptionHandler());
         executorBuilder.setThreadFactory(new JBossThreadFactory(new ThreadGroup("build group"), Boolean.FALSE, null, "build-%t",

--- a/core/launcher/src/main/java/io/quarkus/launcher/JBangIntegration.java
+++ b/core/launcher/src/main/java/io/quarkus/launcher/JBangIntegration.java
@@ -83,7 +83,7 @@ public class JBangIntegration {
                         @Override
                         protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundException {
                             if (name.startsWith("org.") && !(name.startsWith("org.xml.") || name.startsWith("org.w3c.")
-                                    || name.startsWith("org.jboss."))) {
+                                    || (name.startsWith("org.jboss.") && !name.startsWith("org.jboss.logging.")))) {
                                 //jbang has some but not all the maven resolver classes we need on its
                                 //class path. These all start with org. so we filter them out to make sure
                                 //we get a complete class path

--- a/core/runtime/src/main/java/io/quarkus/runtime/ExecutorRecorder.java
+++ b/core/runtime/src/main/java/io/quarkus/runtime/ExecutorRecorder.java
@@ -170,9 +170,12 @@ public class ExecutorRecorder {
         if (threadPoolConfig.queueSize.isPresent()) {
             if (threadPoolConfig.queueSize.getAsInt() < 0) {
                 builder.setMaximumQueueSize(Integer.MAX_VALUE);
+                builder.setQueueLimited(false);
             } else {
                 builder.setMaximumQueueSize(threadPoolConfig.queueSize.getAsInt());
             }
+        } else {
+            builder.setQueueLimited(false);
         }
         builder.setGrowthResistance(threadPoolConfig.growthResistance);
         builder.setKeepAliveTime(threadPoolConfig.keepAliveTime);

--- a/extensions/vertx/runtime/src/main/java/io/quarkus/vertx/core/runtime/QuarkusExecutorFactory.java
+++ b/extensions/vertx/runtime/src/main/java/io/quarkus/vertx/core/runtime/QuarkusExecutorFactory.java
@@ -88,9 +88,12 @@ public class QuarkusExecutorFactory implements ExecutorServiceFactory {
             if (conf.queueSize().isPresent()) {
                 if (conf.queueSize().getAsInt() < 0) {
                     builder.setMaximumQueueSize(Integer.MAX_VALUE);
+                    builder.setQueueLimited(false);
                 } else {
                     builder.setMaximumQueueSize(conf.queueSize().getAsInt());
                 }
+            } else {
+                builder.setQueueLimited(false);
             }
             builder.setGrowthResistance(conf.growthResistance());
             builder.setKeepAliveTime(conf.keepAliveTime());

